todo_app/data/trello_items.py

```diff
@@ -2,33 +2,29 @@
 from datetime import datetime
 import requests
 import os
+import pymongo
 
 def trello_get_items():
```

Comment:
here and elsewhere, this function name is now misleading, as you are using MongoDB as data backend